.github/workflows/run-ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Run CI
+
+on: push
+
+jobs:
+  'run-ci':
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+    # change this to (see https://github.com/ruby/setup-ruby#versioning):
+    # uses: ruby/setup-ruby@v1
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
+      with:
+        ruby-version: 2.5.5
```

Comment:
According to the [README](https://github.com/ruby/setup-ruby#supported-version-syntax), `ruby-version` reads `.ruby-version`, so it can be removed.

```suggestion
```

Reply:
> If the ruby-version input is not specified, .ruby-version is tried first, followed by .tool-versions

Thank you!